cli/configCra.js

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env node
+// @ts-check
+const path = require('path');
+const fs = require('fs');
+// Append current node_modules to the module search path, so require('react-scripts') can work.
+module.paths.push(path.resolve(process.cwd(), './node_modules'));
+
+// 1. Create `jest.config.js`
+try {
+  // @ts-expect-error This is meant to run where react-scripts is installed
+  const createJestConfig = require('react-scripts/scripts/utils/createJestConfig.js');
+  const jestConfig = createJestConfig(
+    (/** @type {string} */ filePath) => path.posix.join('<rootDir>', filePath),
+    null,
+    true,
+  );
+  jestConfig.transform = {
+    '^.+\\.(js|jsx|mjs|cjs|ts|tsx)$':
+      'react-scripts/config/jest/babelTransform.js',
+    '^.+\\.(css|scss|sass)$': 'jest-preview/transforms/css',
+    '^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)':
+      'jest-preview/transforms/fileCRA',
+  };
+  jestConfig.transformIgnorePatterns =
+    jestConfig.transformIgnorePatterns.filter(
+      (/** @type {string} */ pattern) =>
+        pattern !== '^.+\\.module\\.(css|sass|scss)$',
+    );
+  try {
```

Comment:
I agree. However, actually, Since I explained that `jest-preview config-cra` target user with un-ejected app. So we don't even need to use `try catch` for the simplicity of source code.

We can check more strictly when developing `jest-preview config`